### ISSUE TITLE
npm run dev error

### ISSUE THREAD

**Issue by zhilianbi520** [vuejs-templates/webpack-simple#39]
[HMR] Waiting for update signal from WDS...
build.js:1256 Uncaught Error: Cannot find module "./src/main.js"
    at webpackMissingModule (build.js:1256)
    at Object.<anonymous> (build.js:1256)
    at __webpack_require__ (build.js:658)
    at build.js:705
    at build.js:708
webpackMissingModule @ build.js:1256
(anonymous) @ build.js:1256
__webpack_require__ @ build.js:658
(anonymous) @ build.js:705
(anonymous) @ build.js:708
client?20ba:38 [WDS] Hot Module Replacement enabled.
client?20ba:38 [WDS] Errors while compiling.
client?20ba:80 multi main
Module not found: Error: Can't resolve 'babel' in 'F:\myfuture\zbc'
resolve 'babel' in 'F:\myfuture\zbc'
  Parsed request is a module
  using description file: F:\myfuture\zbc\package.json (relative path: .)
  after using description file: F:\myfuture\zbc\package.json (relative path: .)
    resolve as module
      F:\myfuture\node_modules doesn't exist or is not a directory
      F:\node_modules doesn't exist or is not a directory
      looking for modules in F:\myfuture\zbc\node_modules
        using description file: F:\myfuture\zbc\package.json (relative path: ./node_modules)
        after using description file: F:\myfuture\zbc\package.json (relative path: ./node_modules)
          using description file: F:\myfuture\zbc\package.json (relative path: ./node_modules/babel)
            as directory
              F:\myfuture\zbc\node_modules\babel doesn't exist
            no extension
              F:\myfuture\zbc\node_modules\babel doesn't exist
            .js
              F:\myfuture\zbc\node_modules\babel.js doesn't exist
            .json
              F:\myfuture\zbc\node_modules\babel.json doesn't exist
[F:\myfuture\node_modules]
[F:\node_modules]
[F:\myfuture\zbc\node_modules\babel]
[F:\myfuture\zbc\node_modules\babel]
[F:\myfuture\zbc\node_modules\babel.js]
[F:\myfuture\zbc\node_modules\babel.json]
 @ multi main
errors @ client?20ba:80
sock.onmessage @ socket.js?e5d0:37
EventTarget.dispatchEvent @ eventtarget.js?a091:51
(anonymous) @ main.js?75ae:274
SockJS._transportMessage @ main.js?75ae:272
EventEmitter.emit @ emitter.js?d38a:50
WebSocketTransport.ws.onmessage @ websocket.js?5d51:35

执行npm run dev 报错~！ 命令行报Module not found: Error: Can't resolve 'babel' in 'F:\myfuture\zbc'    可是babel已全局安装。

**Comment by li3p** [comment]
rollback webpack to version 2.1.0-beta.25 fix this

```bash
npm install webpack@2.1.0-beta.25 -g
```

**Comment by zhilianbi520** [comment]
@li3p  ok i try again~!thanks~

**Comment by zhilianbi520** [comment]
@li3p   Can't solve this problem~！！

**Comment by zhilianbi520** [comment]
This problem has been solved,thanks~!

**Comment by li3p** [comment]
the better way to solve this is add `-loader` to the name of webpack's loader package. like:

```
			{
				test: /\.js$/,
				loader: 'babel-loader',
				exclude: /node_modules/
			},
```

That wont import the restriction to `webpack` version in your `package.json`。I created a PR, @cnahliu  can you help me review it ?

**Comment by kbqncf** [comment]
many thanks~  
change this code
`{
        test: /\.js$/,
        loader: 'babel',
        exclude: /node_modules/
      }`

to this one
`{
        test: /\.js$/,
        loader: 'babel-loader',
        exclude: /node_modules/
      }`
it works !

**Comment by chrisvfritz** [comment]
Fixed in #40.
